medical_practitioner/views/medical_menu.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    Copyright 2017 Creu Blanca
+    Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <data>
+
+        <menuitem id="medical_conf_practitioner_menu"
```

Comment:
Is there a reason we are creating a top level menu for this? Why not use the pre-existing "Medical Professionals" top level menu that Physicians is currently using?